deepmd/dpmodel/descriptor/se_t_tebd.py

```
@@ -743,3 +757,61 @@
+        return data
+
+    @classmethod
+    def deserialize(cls, data: dict) -> "DescrptSeTTebd":
+        """Deserialize from dict."""
+        data = data.copy()
+        check_version_compatibility(data.pop("@version"), 1, 1)
+        data.pop("@class")
+        data.pop("type")
+        variables = data.pop("@variables")
+        embeddings = data.pop("embeddings")
+        env_mat = data.pop("env_mat")
```

Ruff

Local variable `env_mat` is assigned to but never used

Remove assignment to unused variable `env_mat`

(F841)